compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -251,6 +251,7 @@ OMR::X86::CodeGenerator::initialize(TR::Compilation *comp)
       {
       self()->setUseSSEForSinglePrecision();
       self()->setUseSSEForDoublePrecision();
+      self()->setSupportsAutoSIMD();
```

Comment:
Ok sounds good - happy for it to be added once we are ready to actually attempt some SIMD